encrypted-media.js

```diff
@@ -289,7 +289,7 @@
     'media-resource': { func: videoref_helper, fragment: 'media-resource', link_text: 'media resource',  },
     'media-crossorigin': { func: code_videoref_helper, fragment: 'element-attrdef-media-crossorigin', link_text: 'crossorigin',  },
     'mime-types': { func: videoref_helper, fragment: 'mime-types', link_text: 'MIME types',  },
-    'valid-mime-type': { func: infrastructure_helper, fragment: 'valid-mime-type', link_text: 'valid media MIME type',  },
+    'valid-mime-type': { func: infrastructure_helper, fragment: 'valid-mime-type', link_text: 'valid MIME type',  },
```

Comment:
The infrastructure helper plus the fragment build a link to HTML5.1:  https://www.w3.org/TR/html51/infrastructure.html#valid-mime-type